src/react/components/pages/editorPage/canvas.test.tsx

```diff
@@ -58,7 +58,7 @@ describe("Editor Canvas", () => {
         const editorMock = Editor as any;
         editorMock.prototype.addContentSource = jest.fn(() => Promise.resolve());
         editorMock.prototype.scaleRegionToSourceSize = jest.fn((regionData: any) => regionData);
-        editorMock.prototype.RM = new RegionsManager(null, null, null);
+        editorMock.prototype.RM = new RegionsManager(null, null);
```

Comment:
Did you update canvas tools?  I don't see a change to package.json, but i do see a change to some of these method calls...?

Reply:
Canvastools ended up getting updated at some point, didn't add the regionmoveendbit, that's another pr